### ISSUE TITLE
Remove Progressbar

### ISSUE THREAD

**Issue by mwaeckerlin** [MetaMask/metamask-onboarding#8]
Thank you for the tool, it's quite helpful. The only thing that bothers me is a «progressbar» that appears on top of the HTML-body when I use it. If I uncomment the MetamaskOnboarding-stuf, the progressbar disappears, so it seems to come from this code here.

Please remove that progressbar, or at least make it optional.

On the other side, if there is any progress information that can be displayed, please provide it to a callback, so that the application can handle the rendering.

See red arrow that points to this «progressbar», the growing black line on white background:

![image](https://user-images.githubusercontent.com/4056726/71556471-6c7f3580-2a39-11ea-9926-097c69e697f1.png)

**Comment by mwaeckerlin** [comment]
I found the source of the problem, it's the `iframe`s of the `FORWARDER_ID`:
```
<iframe src="https://fwd.metamask.io" id="FORWARDER_ID" width="0" height="0"></iframe>
```

![image](https://user-images.githubusercontent.com/4056726/71557052-e36bfc80-2a40-11ea-8f5b-dca2bbfe3b42.png)

Workaround in CSS:
```
iframe {
  display: none;
}
```

**Comment by whymarrh** [comment]
@mwaeckerlin thanks for the suggestion, we should definitely be able to make it optional

**Comment by Gudahtt** [comment]
It looks like this isn't a _progress bar_, it's a border. There's a border on the `iframe` by default.

Thanks for the report! This will be removed.

**Comment by Gudahtt** [comment]
Also, I noticed that you have multiple instances of the forwarder `iframe` in that screenshot. That was a bug - it should be fixed by #25. Please let us know if you see that again.
